src/tests/locale/test_spelling.py

```diff
@@ -0,0 +1,98 @@
+import polib
+import enchant as ec
+import os
+from enchant.checker import SpellChecker
+from enchant.tokenize import Filter, HTMLChunker
+from shutil import rmtree
+
+class PythonFormatFilter(Filter):
+    def _skip(self, word):
+        if word[:2] == "%(":
+            return True
+        if word[0] == "{":
+            return True
+        return False
+
+# The languages dict maps the language of the po-file to the the dictionary language used by pyenchant
+languages = {
+        "de": "de_DE",
+        "en": "en_US",
+}
+
+# These wordlists in the IGNORES_DIR only need to contain uncapitalized words,
+# the corresponding capitalized words are automatically recognized.
+LOCALES_DIR = os.path.abspath('../../pretix/locale')
+IGNORES_DIR = os.getcwd()
+BUILD_DIR = os.path.abspath('./_build')
+
+try:
+    print('Creating build directory at' + BUILD_DIR + '…')
+    os.mkdir(BUILD_DIR)
+except FileExistsError:
+    print("File or directory" + BUILD_DIR + "already exists, deleting…")
+    rmtree(BUILD_DIR)
```

Comment:
This should still be able to fail due to the build-directory not being deletable (because it is owned by another user, for example). One could take care of that exception, or just assume that this case never happens...
However, if you do this in a filesystem where you are only allowed to add, but not to remove, this might crash nevertheless…
Depends on how many special cases you want to cover.

Reply:
I personally don't think that we need to cover too many special cases, I'd cover them if they appear. For the moment, checking for an existing directory should be enough.
However, I'll leave this to @raphaelm for decision ;)